src/components/PreviewCard/__snapshots__/PreviewCard.test.tsx.snap

```diff
@@ -116,7 +116,6 @@ exports[`<PreviewCard /> when loading matches its snapshot 1`] = `
       >
         <Spinner
           color="metadata"
-          hideText={false}
```

Comment:
This was intentional?

Reply:
yes, it just removed the default `false` value. still resolves to false